tailwind.js

```diff
@@ -778,12 +778,12 @@ module.exports = {
   modules: {
     appearance: ['responsive'],
     backgroundAttachment: ['responsive'],
-    backgroundColors: ['responsive', 'hover', 'focus', 'group-hover'],
+    backgroundColors: ['responsive', 'hover', 'focus', 'focus-within', 'group-hover'],
```

Comment:
I think we should remove `focus-within` from the styles we are not using, just to keep the css smaller and with only the things we need.